colorclusters/image_utils.py

```diff
@@ -35,7 +35,7 @@ def map_index_to_paletted_image(size, index_data, colors):
     return palette_image
 
 
-def map_to_paletted_image(img, colors, distance=euclidean):
+def map_to_paletted_image(img, colors, output_queue, distance=euclidean):
```

Comment:
Pre-existing code that explicitly states the distance function will now misinterpret it as being an output queue.
Can be fixed by switching it to:
... , distance=euclidean, output_queue=None):